metabase/metabase
# Copy question link from dashboard

Closes #41070

### Description

Makes titles of most dashboard charts actual HTML anchors.
Thanks to this users can right-click the title to copy URL, middle-click it to open it in a new tab, drag & drop it to browser's address bar, etc.

### How to verify

1. Create a dashboard
2. Add various visualizations to it
3. Right-click visualization title 

Context menu should open and you should be able to copy the URL of the link.

### Demo


https://github.com/metabase/metabase/assets/6830683/b31a0664-7367-4fd0-b6bc-9ce5a21eed1c
